src/pages/accounts/AdminAccountsPage.slice.ts

```diff
@@ -0,0 +1,70 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
```

Comment:
I recommend writing some tests for this and whatever else you can.

Reply:
I will probably have tests when I do requests the backend. Right now, I do not see anything worth testing. Only the reducers that are 1 line long are being used. Like `setSelectedAccount`